backend/app/controllers/spree/admin/resource_controller.rb

```diff
@@ -134,9 +134,15 @@ def belongs_to(model_name, options = {})
     end
   end
 
+  def nil_parent?(options = {})
```

Comment:
This is confusing because the convention in Ruby is that methods that end in question marks (known as predicate methods) return boolean values.

Reply:
Ah I see, it definitely needed to be changed then. It's changed to `resource_error_filter` now